mysql-test/suite/funcs_1/r/is_columns_is.result

```diff
@@ -418,24 +418,24 @@ def	information_schema	SLAVE_STATUS	Master_SSL_Key	34	NULL	YES	varchar	512	1536
-def	information_schema	SLAVE_STATUS	Replicate_Ignore_DB	16	NULL	NO	varchar	64	192	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(64)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Ignore_Domain_Ids	48	NULL	NO	varchar	512	1536	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(512)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Ignore_Server_Ids	41	NULL	NO	varchar	512	1536	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(512)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Ignore_Table	18	NULL	NO	varchar	64	192	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(64)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Rewrite_DB	56	NULL	NO	varchar	1024	3072	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(1024)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Wild_Do_Table	19	NULL	NO	varchar	64	192	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(64)			select		NEVER	NULL	NO	NO
-def	information_schema	SLAVE_STATUS	Replicate_Wild_Ignore_Table	20	NULL	NO	varchar	64	192	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(64)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Do_DB	15	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Do_Domain_Ids	47	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Do_Table	17	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Ignore_DB	16	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Ignore_Domain_Ids	48	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Ignore_Server_Ids	41	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Ignore_Table	18	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Rewrite_DB	56	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Wild_Do_Table	19	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
+def	information_schema	SLAVE_STATUS	Replicate_Wild_Ignore_Table	20	NULL	NO	varchar	21844	65532	NULL	NULL	NULL	utf8mb3	utf8mb3_general_ci	varchar(21844)			select		NEVER	NULL	NO	NO
```

Comment:
Actually @bnestere do these count as tests for this issue?

Reply:
Sometimes they will, but in this case I'd say it isn't sufficient, as the existing tests weren't enough to catch the regression in the first place :sweat_smile:. So we should add tests to ensure validly-long values don't get truncated.

Reply:
I reviewed the SHOW REPLICA STATUS listing and gave them a good consideration.
(Some like these `Replicate_`s come straight from user configs, others are computed live or are string messages (the L10N-able kind).)

I’ve decided that edge case testing should be part of their feature testing; that is, verify their SHOW REPLICA STATUS entry in addition to their `SELECT GLOBAL/SESSION`/`SHOW STATUS`/etc.
Which means they ideally should be available on the branch SHOW REPLICA STATUS added them; i.e., ideally backport to the earliest applicable branches.

The maintenance sprint is finishing up, @bnestere. Lemme see if tests for these `Replicate_`s ones can make it.

Reply:
filed [MDEV-35948](https://jira.mariadb.org/browse/MDEV-35948) to generalize the scope to all of SHOW REPLICA STATUS

Reply:
>  i.e., ideally backport to the earliest applicable branches.
...
> filed [MDEV-35948](https://jira.mariadb.org/browse/MDEV-35948) to generalize the scope to all of SHOW REPLICA STATUS

Sure. It makes sense to have the complete test in 10.5+ as well, though with the addition...

> Lemme see if tests for MDEV-35693’s Replicate_s ones can make it.

I think would be good. I'd say we should add tests now for any of the fields you've changed in this patch, and then backport it and test the rest in 10.5 (say during the next light week). And the cutoff for this part is Friday, so I'd think there's time (this would take priority over MDEV-35304).

Reply:
I’ve prepared #3795 for the ones reported (and are mainly affected) by MDEV-35693.
I’ve yet to search where the rests’ (both the latter `Replicate_`s and others) tests are.